Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -400,7 +406,9 @@ private IAuthenticationProvider[] GetAuthenticationProviders(User user)
 
             if (providers.Length == 0)
             {
-                providers = new IAuthenticationProvider[] { _defaultAuthenticationProvider };
```

Comment:
Afaik the `_defaultPasswordResetProvider` field is now unused and can be removed.

Reply:
Where is this one used?

Reply:
Line 429-432 I suppose?

Reply:
Actually, I can remove like 429-432 no issue, but the "_defaultPasswordResetProvider" is used grossly on line 150 to I guess get the type? So I can't remove that.

Reply:
All good!